cdc/kv/regionlock/region_range_lock.go

```diff
@@ -218,7 +324,7 @@ func (l *RegionRangeLock) tryLockRange(startKey, endKey []byte, regionID, versio
 	overlappingEntries := l.getOverlappedEntries(startKey, endKey, regionID)
 
 	if len(overlappingEntries) == 0 {
-		checkpointTs := l.rangeCheckpointTs.GetMin(startKey, endKey)
+		checkpointTs := l.rangeCheckpointTs.getMinTsInRange(startKey, endKey)
```

Comment:
I wonder why it is called `rangeCheckpointTs` instead of `rangeResolvedTs`.

Reply:
It should be `ResolvedTs`. `CheckpointTs` is a mistake and we follow it in a long time.